lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -294,7 +294,7 @@ protected function _getPortableSequenceDefinition($sequence)
             $sequence += $data;
         }
 
-        return new Sequence($sequenceName, $sequence['increment_by'], $sequence['min_value']);
+        return new Sequence($sequenceName, (int) $sequence['increment_by'], (int) $sequence['min_value']);
```

Comment:
Should we update the signature of `Sequence::__construct()` instead and declare them integers? With the absence of strict scalar types on the calling end, the effect will be the same.

Reply:
Didn't notice the absence of strict declaration. Will do.

Reply:
Shouldn't I leave there `return new Sequence($sequenceName, (int) $sequence['increment_by'], (int) $sequence['min_value']);` as well? It will have to be added later anyway I suppose.

Reply:
Probably. I'd start with a test.

Reply:
I think `(int)` conversion should stay and adding type hints would be nice. Done